tasks/27_basic-subclasses/basic-subclasses.js

```diff
@@ -0,0 +1,21 @@
+export class Human {
+  constructor(name, sex) {
+    this.name = name;
+    this.sex = sex;
+  }
+}
+export class Woman extends Human {
+  constructor(name, sex) {
+    super(name, sex);
+  }
+}
+export class Man extends Human {
+  constructor(name, sex) {
+    super(name, sex);
+  }
+}
```

Comment:
Seems like you have to identical classes named differently

Reply:
That's because in the task description it says that Man and Women should extend the `Human` class, but the description doesn't give any additional properties for `Man` or `Women`.